packages/domain/src/use-cases/transactions/SupportedTransactionRequest.ts

```diff
@@ -24,3 +25,17 @@ export type SupportedTransactionRequest =
   | UpdateFollowPolicyRequest
   | UpdateProfileDetailsRequest
   | UpdateProfileImageRequest;
+
+export const ProtocolCallKinds = [
+  TransactionKind.COLLECT_PUBLICATION,
+  TransactionKind.CREATE_COMMENT,
+  TransactionKind.CREATE_POST,
+  TransactionKind.CREATE_PROFILE,
+  TransactionKind.FOLLOW_PROFILES,
+  TransactionKind.MIRROR_PUBLICATION,
+  TransactionKind.UPDATE_PROFILE_IMAGE,
+  TransactionKind.UNFOLLOW_PROFILE,
+  TransactionKind.UPDATE_PROFILE_DETAILS,
+  TransactionKind.UPDATE_FOLLOW_POLICY,
+  TransactionKind.UPDATE_DISPATCHER_CONFIG,
+];
```

Comment:
I would like to generate a union out of this so to type more correctly the places that currently accepts `SupportedTransactionRequest` giving the wrong message that all Lens SDK supported tx can be: meta-tx, native-tx, proxy-actions, etc).

I might do this as initial refactoring for DA, cause that's another subset of the supported tx.